site/src/pages/WorkspacePage/WorkspaceTopbar/WorkspaceTopbar.tsx

```diff
@@ -120,58 +120,77 @@ export const WorkspaceTopbar = (props: WorkspaceProps) => {
         }}
       >
         <TopbarData>
-          <TopbarAvatar src={workspace.template_icon} />
-          <span css={{ fontWeight: 500 }}>{workspace.name}</span>
+          <TopbarIcon>
+            <PersonOutline />
+          </TopbarIcon>
+          <Tooltip title="Owner">
+            <span>{workspace.owner_name}</span>
+          </Tooltip>
           <TopbarDivider />
-          <Link
-            component={RouterLink}
-            to={`/templates/${workspace.template_name}`}
-            css={{ color: "inherit" }}
+          <Popover
+            mode="hover"
+            // title={`${
+            //   workspace.template_display_name ?? workspace.template_name
+            // } on version ${workspace.latest_build.template_version_name}`}
```

Comment:
Is the commented code intentional? Will it be used later?

Reply:
No, ty!